corehq/apps/linked_domain/templates/linked_domain/tabs/pull_release_content_tab.html

```diff
@@ -5,7 +5,7 @@
   <div data-bind="if: domainLink">
     <p>
       {% blocktrans %}
-        This project is linked to the upstream project <a data-bind="attr: {'href': domainLink.upstreamUrl}, text: domainLink.master_domain"></a>.<br/>
+        This project is linked to the upstream project <a data-bind="attr: {'href': domainLink.upstreamUrl}, text: domainLink.upstreamDomain"></a>.<br/>
```

Comment:
Is this change supposed to be part of this PR? I looked at the version of `domain_links.js` that's on the `gh/improve-erm-ui` which is [this](https://github.com/dimagi/commcare-hq/blob/e83c1984be750d0499a7dd4ddd8be9ea94a3013b/corehq/apps/linked_domain/static/linked_domain/js/domain_links.js), and it still has `master_domain`.

Reply:
Good catch. I messed up a conflict resolution and thought I had taken care of it, but obviously not.